src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -1700,7 +1700,7 @@ class TopToolbar final : public Window
 
         auto screenPos = sourceScreenPos;
         uint16_t maxPossibleHeight = ZoomLevel::max().ApplyTo(
-            std::numeric_limits<decltype(TileElement::BaseHeight)>::max() - 32);
+            std::numeric_limits<decltype(TileElement::BaseHeight)>::max() - 1);
```

Comment:
Is this subtraction really necessary? The scenery entry's height is already subtracted a few lines down. I guess it may still be needed because IIRC the max base height is a way the game marks elements to be removed. Not sure if that's still the case.

Reply:
Do you mean the -1, or the -32?
I thought I had already commented on this, but I don't any comment of mine anywhere.

The -32 is not needed, the -1 is, without it if you place something at the top level, I'm not even sure exactly what happens, but not what you'd want.